content-src/components/Spotlight/Spotlight.scss

```diff
@@ -91,10 +91,59 @@
     overflow: hidden;
   }
 
+  .icon-pocket {
```

Comment:
would you be able to add this to `styles/icons.scss` and extend here if necessary?